### DISCUSSION TITLE
Icon request: icon-biceps

### DISCUSSION THREAD

**Discussion by TysGibbons** [FortAwesome/Font-Awesome#21131] [Icon Requests]
I would like to request a muscle icon. This would be the traditional flexed bi-cep as you can see here: http://comps.fotosearch.com/comp/SUE/SUE104/flexing-muscle_~BWBW1165.jpg.

Thank you.

**Comment by JulieKuehl** [comment]
+1

**Comment by shadargee1982** [comment]
+1

**Comment by airstep** [comment]
+1

**Comment by Tonio31** [comment]
+1

**Comment by darkwood-studios** [comment]
+1

**Comment by gbradley** [comment]
Would be great to get this addition.

**Comment by hsvandreas** [comment]
Wow, actually surprised that this icon doesn't exist yet. Big +1

**Comment by totoprayogo1916** [comment]
+1 👍

**Comment by CalumChilds** [comment]
+1

**Comment by feerposser** [comment]
+1 🦾

**Comment by manudefrutosvila** [comment]
💪

**Comment by FredM7** [comment]
Yes ditto, +1

**Comment by gerardreches** [comment]
Still doesn't exist? :(

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
💪 [U+1F4AA](https://emojipedia.org/emoji/%F0%9F%92%AA/) Flexed Biceps

**Comment by tskxz** [comment]
After 9 years, stil doesn't exists

**Comment by pierfreeman** [comment]
+1

**Comment by QINGCHARLES** [comment]
10 years now😂

**Comment by namik-gamedev** [comment]
+1 :100:

**Comment by janklan** [comment]
Can we please have some muscles over here? 💪

**Comment by feerposser** [comment]
Eleven years right now haha
Since my comment I already change from web dev. to game dev. and this icon still not implemented

**Comment by iveyopt** [comment]
+1, I keep thinking this icon must already exist

**Comment by gerardreches** [comment]
This icon is a must

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by tskxz** [comment]
After 11 years, finally, the icon will be released :smile:

**Comment by gbradley** [comment]
This will be the end of an era

**Comment by sekurum** [comment]
Well, it's 2025 now, still no muscle?

**Comment by janklan** [comment]
I bet the muscle depiction has been copyrighted by Chuck Norris for the last four of five eons.

**Comment by umarbutler** [comment]
+1

**Comment by ividrine** [comment]
+1, need this to exist in 2025!

**Comment by nickjbedford** [comment]
+1 I'm so surprised this still doesn't exist, despite there being numerous sports and fitness icons.

**Comment by shinmorisawa** [comment]
+1 where is it exactly?

**Comment by sensibleworld** [comment]
Added in 7.2: https://fontawesome.com/icons/bicep?f=classic&s=solid
